.github/workflows/CI.yml

```diff
@@ -10,6 +10,7 @@ on:
 jobs:
   build:
     runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.os == 'ubuntu-latest' }}
```

Comment:
In my opinion if ubuntu build fails CI build should fail too.

Reply:
This setting change just makes it so that when linux fails, the other jobs won't get cancelled. The job will still fail the CI build.

Linux builds continually have different implementation for code formatting errors. :( Namely formatting of comments.
This change makes it so that when it does fail, the Windows build and tests will also continue to completion. Allowing for getting more information about any failures with the same commit.